session-replacement/src/main/java/com/amadeus/session/servlet/CookieSessionTracking.java

```diff
@@ -98,14 +98,17 @@ public void propagateSession(RequestWithSession request, Object response) {
   }
 
   private String cookiePath(HttpServletRequest request) {
-    return contextPath ? request.getContextPath() + "/" : "/";
+    if (contextPath != null && !contextPath.isEmpty()) {
+      return contextPath;
+    }
+    return request.getContextPath();
   }
 
   @Override
   public void configure(SessionConfiguration conf) {
     super.configure(conf);
     httpOnly = Boolean.valueOf(conf.getAttribute(COOKIE_HTTP_ONLY_PARAMETER, "true"));
     secure = Boolean.valueOf(conf.getAttribute(SECURE_COOKIE_PARAMETER, "false"));
-    contextPath = Boolean.valueOf(conf.getAttribute(COOKIE_CONTEXT_PATH_PARAMETER, "true"));
+    contextPath = conf.getAttribute(COOKIE_CONTEXT_PATH_PARAMETER, "");
```

Comment:
Let's make `null` as default value. It will make test in CookieSessionTracking little bit faster.